.github/workflows/markdown-lint-fix.yml

```
@@ -30,6 +30,7 @@ jobs:
         run: |
           yarn fix:md
           yarn fix:fm
+          yarn content fix-flaws
```

```suggestion
          yarn content fix-flaws
          yarn fix:md
          yarn fix:fm
```
Let's put `content fix-flaws` first. As it has no chance of failing we'll get maximum fixing done before any linter exits with error.